Added/Improved Intros for the Map Topics (#6887)
github/docs: content/github/getting-started-with-github/signing-up-for-github/index.md

---
title: Signing up for GitHub

> intro: 'Sign up with your Email to Get started with setting up a new {% data variables.product.prodname_dotcom %} account.'

Reviewer: ```suggestion
intro: 'Start using {% data variables.product.prodname_dotcom %} for yourself or your team.'
```
